build-system/babel-plugins/babel-plugin-transform-json-configuration/test/fixtures/transform/transform-internal-runtime-version-call/input.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+jsonConfiguration({});
```

Comment:
Should we just bake this into `dict()`?

Reply:
I mean, can we tell when the JS object parameter has variable refs? If so, can the transform detect and automatically do the right thing for static JSON?

That would lift the burden of knowing when to use `dict` vs `jsonConfiguration` off developers' shoulders. 

If that's not possible, how about a more snazzy name to encourage usage? E.g. `fastJson` 😛